packages/graphing/src/container/index.jsx

```diff
@@ -17,7 +17,7 @@ const mapDispatchToProps = dispatch => ({
   onChangeMarks: m => dispatch(changeMarks(m)),
   onUndo: () => dispatch(ActionCreators.undo()),
   onRedo: () => dispatch(ActionCreators.redo()),
-  onReset: () => dispatch(ActionCreators.jumpToPast(0))
```

Comment:
I'm afraid what `ActionCreators.reset` does is more than just changing marks to empty array. Perhaps there's more to it - worth a check. I'm not familiar with this, just thinking out loud.